lib/active_merchant/billing/gateways/hps.rb

```diff
@@ -296,21 +337,22 @@ def commit(action, &request)
         )
       end
 
+      SUCCESSFUL_RESPONSE_CODES = %w(0 00 85)
```

Comment:
Where'd the `0` come from? I see where the `00` and `85` were before.

Reply:
Because of reasons, 0 is the successful return code for a check transaction -- https://github.com/activemerchant/active_merchant/pull/3500/files/675919e04c8c71b93cd524e2a7a5b7c50bc11c25#diff-cc2188012eda9d8edc721798125c486fR505 is a sample of such a response.